bpdm-gate-api/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/api/model/response/ChangelogResponse.kt

```diff
@@ -17,19 +18,17 @@
  * SPDX-License-Identifier: Apache-2.0
  ******************************************************************************/
 
-package org.eclipse.tractusx.bpdm.gate.dto.response
+package org.eclipse.tractusx.bpdm.gate.api.model.response
 
 import io.swagger.v3.oas.annotations.media.Schema
+import java.time.Instant
 
-@Schema(description = "Contains overall result of a sharing process validation request")
-data class ValidationResponse(
-    @Schema(description = "Overall status of the result of the validation")
-    val status: ValidationStatus,
-    @Schema(description = "All found validation errors of this record")
-    val errors: Collection<String>
+@Schema(name = "ChangelogResponse", description = "Changelog entry for a business partner")
+data class ChangelogResponse(
+    @Schema(description = "External ID of the changelog entry")
+    val externalId: String,
+    @Schema(description = "The type of the change")
+    val businessPartnerType: LsaType,
+    @Schema(description = "The timestamp of the operation")
+    val createdAt: Instant
```

Comment:
Field name createdAt is a bit misleading here. The changelog entry should state when a business partner has been updated. In this regard better name the field soemthing like "modifiedAt", "updatedAt" or similar.